selfdrive/athena/athenad.py

```diff
@@ -472,6 +474,33 @@ def log_handler(end_event):
       cloudlog.exception("athena.log_handler.exception")
 
 
+def stat_handler(end_event):
+  while not end_event.is_set():
+    last_scan = 0
+    try:
+      curr_scan = sec_since_boot()
+      if curr_scan - last_scan > 10:
+        for stat_entry in os.listdir(STATS_DIR):
+          stat_path = os.path.join(STATS_DIR, stat_entry)
+          with open(stat_path) as f:
+            stats = json.loads(f.read())
```

Comment:
```suggestion
          with open(stat_path) as f:
            stats = json.load(f)
```

Rest of the code can be outside of the context manager.